content/nginxaas-azure/troubleshooting/guide.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshooting guide
+weight: 50
+toc: true
+url: /nginxaas/azure/troubleshooting/guide/
+type:
+- how-to
+--- 
+
+Many users encounter difficulties getting started with NGINXaaS for Azure, or have difficulty debugging performance or run-time problems. Here is some guidance about how to resolve common problems. 
+
+## Certificate health ##
+
+Certificates are a common source of confusion and errors. It is important to understand that certificates are integral to your deployment's health. Once you have added a certificate as a resource to your deployment, our service will always attempt to fetch that certificate when applying your NGINX configuration. It doesn't matter whether the certificate is referenced by the NGINX configuration or not: we will always try to fetch it as long as it is added to the deployment. 
+
+- Ensure that all certificate resources are needed by the deployment and that they are not about to expire. 
+- If a certificate on the deployment expires in your Azure Key Vault, we will not be able to fetch it and your deployment will enter a degraded state. Our service will then be unable to apply the latest software updates to your dataplane. 
```

Comment:
based on our last customer interaction, maybe change this to:
... enter a degraded state and may not be able to serve traffic.